scripts/first-login/001-setup-prod.sh

```diff
@@ -40,7 +40,8 @@ After=systemd-user-sessions.service
 Type=simple
 Environment="MEILI_SERVER_PROVIDER=$MEILISEARCH_SERVER_PROVIDER"
 Environment="MEILI_DUMP_DIR=$MEILI_DUMP_DIR"
-ExecStart=/usr/bin/meilisearch --db-path /var/lib/meilisearch/data.ms --env $MEILISEARCH_ENVIRONMENT --dump-dir $MEILI_DUMP_DIR
+Environment="MEILI_DB_PATH=$MEILI_DB_PATH"
```

Comment:
I think to prevent breaking for existing users it would be needed to set a default value here, no?:

https://github.com/meilisearch/cloud-providers/blob/26ad0b47c59e60bcc0e53d768262f1e71bb14a14/scripts/first-login/000-set-meili-env.sh#L17-L24

Reply:
Yes. My bad. Should I add?

Reply:
Yes :)